optimize_versioned_docs/version-3.13.0/apis-tools/optimize-api/tutorial.md

```diff
@@ -0,0 +1,167 @@
+---
+id: optimize-api-tutorial
+title: Tutorial
+description: "Step through an example to view your existing clients, create a client, view a particular client's details, and delete a client."
+---
+
+In this tutorial, we'll step through examples to highlight the capabilities of the Optimize API, such as listing your existing dashboard IDs, or deleting a dashboard.
+
+## Getting started
```

Comment:
The flow here is a bit odd. I would swap these sections (Getting started and Prerequisites) because I can't go the steps in Getting started until I have a cluster. 

Because I already had a cluster, I did these steps immediately.

Reply:
Agreed. Resolved in latest commit 👍